recipes/scikit-video/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "sk-video" %}
+{% set version = "1.1.9" %}
+{% set sha256 = "812779b5245e9eaa330ba04fa2bc050b11f1d17246050da4365386d993108c49" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - pip
+    - numpy
+    - scipy
+  run:
+    - python
+    - numpy
+    - scipy
+    - ffmpeg
+
+test:
+  imports:
+    - skvideo
+    - skvideo.io
+    - skvideo.motion
+    - skvideo.measure
+    - skvideo.datasets
+    - skvideo.utils
+
+about:
+  home: https://github.com/scikit-video/scikit-video
+  license: BSD-3-Clause
+  license_family: BSD
```

Comment:
Please raise an issue upstream to include the license file with the source distribution, and then include the url to the issue as a comment here. Then manually include the license as in:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

Reply:
thanks to the package maintainer, we bumped a new version with the license included in the source distributionl